pkg/asset/ignition/machine/node.go

```diff
@@ -4,12 +4,27 @@ import (
 	"fmt"
 	"net/url"
 
+	//"github.com/apparentlymart/go-cidr/cidr"
 	ignition "github.com/coreos/ignition/config/v2_2/types"
 	"github.com/vincent-petithory/dataurl"
 
 	"github.com/openshift/installer/pkg/types"
+	openstacktypes "github.com/openshift/installer/pkg/types/openstack"
 )
 
+func ignitionURL(installConfig *types.InstallConfig) string {
+	switch installConfig.Platform.Name() {
+	case openstacktypes.Name:
```

Comment:
`baremetal` needs to do something similar - seen https://github.com/openshift/installer/pull/1873/files#diff-d7037682a68150f4dbbbf5cc0d6e8f54R17

Reply:
Yes, that PR is linked from the commit message here (as is the oVirt's equivalent: #1948).

The intention here is to merge this small patch so that all three platforms (baremetal, ovirt, openstack) can validate this is an acceptable solution and that they can build on.